src/display/api.js

```diff
@@ -2447,6 +2473,16 @@ class WorkerTransport {
     return this.messageHandler.sendWithPromise("GetOutline", null);
   }
 
+  getOptionalContentConfig() {
+    return this.messageHandler
+      .sendWithPromise("GetOptionalContentConfig", null)
+      .then(results => {
+        const config = new OptionalContentConfig();
```

Comment:
Just wondering: Any particular reason to not simply pass the `results` when creating the `OptionalContentConfig` instance, and thus essentially inline the `initialize` method in the constructor?

Reply:
I had plans of doing some unit testing this differently. Not going to do that now, so I'll merge it.